Makefile.mac.mk

```diff
@@ -329,6 +333,7 @@ build/frida-mac-universal/lib/$(PYTHON_NAME)/site-packages/frida: build/tmp-mac-
 	mkdir -p $(@D)
 	cp -a build/frida-mac-x86_64/lib/$(PYTHON_NAME)/site-packages/frida $@
 	@touch $@
+
```

Comment:
Shouldn't be any newline here; it's to make it very clear that the targets are corresponding, but for different platforms/archs.